script/install.sh

```diff
@@ -540,13 +574,17 @@ download_and_unpack() {
 }
 
 verify() {
+    local keyring_mirrors=(
+        "https://dlang.org/d-keyring.gpg"
+        "https://github.com/dlang/dlang.org/raw/846269e913114fd77af0cc1aff67269046b70db8/d-keyring.gpg"
```

Comment:
I couldn't find another mirror for this.

Reply:
There you go, https://nightlies.dlang.org/d-keyring.gpg.
In fact you can upload files to sebastian@nightlies.dlang.org:/var/www/builds yourself.
But you cannot move or delete my entries in that folder (sticky bit). Allowing any nginx group member to mess with the install script and the signature key sounds like disaster.